pandas/core/generic.py

```diff
@@ -7538,16 +7538,21 @@ def between_time(
         Parameters
         ----------
         start_time : datetime.time or str
+                The first time value.
         end_time : datetime.time or str
+                The second time value.
         include_start : bool, default True
+                Adding start_time value in the result.
```

Comment:
Looks good, but I would rephrase to something like "whether the start time needs to be included in the result".

I find a bit confusing when you say "Adding", since you're telling users here what they should provide in this parameter.